t/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.5"
 services:
   alpha1:
-    image: dgraph/dgraph:local
+    image: dgraph/dgraph:${DGRAPH_VERSION:-latest}
```

Comment:
if `DGRAPH_VERSION` is not specified in the environment, then we default to latest (doc ref [here](https://docs.docker.com/compose/compose-file/compose-file-v3/)).  This allows developer to run tests using latest tag (or any tag) without building the image locally first.  In CI we set `DGRAPH_VERSION=local` as before.